src/components/@shared/atoms/Lists/index.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import { ListItem, ListItemProps } from '@shared/atoms/Lists'
+
+export default {
+  title: 'Component/@shared/atoms/Lists',
+  component: ListItem
+} as ComponentMeta<typeof ListItem>
+
+const Template: ComponentStory<typeof ListItem> = (args) => (
+  <ListItem {...args} />
+)
+
+interface Props {
+  args: ListItemProps
+}
+
+export const Default: Props = Template.bind({})
+Default.args = {
+  children: (
+    <>
+      <a>List Item</a>
```

Comment:
all technically correct as we have a really weird list component here. Let's see if we can get this more useful for real life. So one of the things we usually want with lists, is to make sure the spacing between the list items stays as we want it to be. And for this we simply need multiple list items. So from pure markup perspective we should have something like this output in the stories here:


```html
<ul>
  <li>List item short</li>
  <li>List item long ipsum dolor sit amet, consectetur adipiscing elit. Mauris aliquam facilisis molestie</li>
  <li>List item long ipsum dolor sit amet, <a href="#">consectetur adipiscing elit</a></li>
</ul>
```

And then the same again for a `ol`. That somehow should be possible with storybook instead of just working with those `args`